axon_sql.go

```diff
@@ -168,7 +169,7 @@ func (a *Axon) insertRow(sourceDB *sqlx.DB, targetDB *sqlx.DB, change *Changeset
 
 	// NOTE: row insert/update/delete logs have been updated to be the same length
 	// to align timestamps to ease viewing.
-	a.Logger.Printf("row insert: %s", change)
+	a.Logger.WithField("time-delta", time.Now().Sub(change.Timestamp)).Printf("row insert: %s", change)
```

Comment:
nit: "Lag" is more meaningful than time-delta

Reply:
Same same!